BE/src/main/resources/schema.sql

```diff
@@ -0,0 +1,86 @@
+DROP TABLE IF EXISTS member;
+CREATE TABLE member
+(
+    member_id BIGINT AUTO_INCREMENT NOT NULL,
+    email     VARCHAR(255) NOT NULL,
+    password  VARCHAR(255) NOT NULL,
+    PRIMARY KEY (member_id)
+);
+
+DROP TABLE IF EXISTS order_sheet;
+CREATE TABLE order_sheet
+(
+    order_id    BIGINT AUTO_INCREMENT NOT NULL,
+    banchan_id  BIGINT NOT NULL,
+    member_id   BIGINT NOT NULL,
+    quantity    INTEGER NOT NULL,
+    total_price INTEGER NOT NULL,
+    PRIMARY KEY (order_id),
+    FOREIGN KEY (banchan_id) REFERENCES banchans (banchan_id),
+    FOREIGN KEY (member_id)  REFERENCES member (member_id)
+);
+
+DROP TABLE IF EXISTS banchans;
+CREATE TABLE banchans
+(
+     banchan_id   BIGINT AUTO_INCREMENT NOT NULL,
+     category_id  INTEGER NOT NULL,
```

Comment:
p2; f.k 이니 타입 변경 필요해보이네요.

```suggestion
     category_id  BIGINT NOT NULL,
```

Reply:
넵 FK타입에 맞게 BIGINT로 수정하였습니다. 🙆‍♂️